table-walkthrough/pom.xml

```
@@ -72,9 +72,15 @@ under the License.
             <version>${flink.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+           <groupId>org.apache.flink</groupId>
+           <artifactId>flink-table-planner_${scala.binary.version}</artifactId>
+           <version>${flink.version}</version>
+           <scope>test</scope>
+       </dependency>
```

Try this instead -- I think it should work. We want to encourage using the scala-free versions wherever possible.

```suggestion
        <dependency>
           <groupId>org.apache.flink</groupId>
           <artifactId>flink-table-planner-loader</artifactId>
           <version>${flink.version}</version>
           <scope>provided</scope>
        </dependency>
        <dependency>
            <groupId>org.apache.flink</groupId>
            <artifactId>flink-table-runtime</artifactId>
            <version>${flink.version}</version>
            <scope>provided</scope>
        </dependency>
```

---

this works as well. thanks for the clarification